### PR TITLE
fix: experts page layout changed for smaller screens

### PR DESCRIPTION
**Pull request by Hallidayo**
## What kind of change does this PR introduce?

Supabase website changed

## What is the current behavior?

If you go onto the [experts](https://supabase.com/partners/experts) page on the supabase website on a mobile device/tablet  the layout of the content stays the same at a large breakpoint:

Screenshot 2022-05-10 at 10 07 24

## What is the new behavior?

I've changed the layout to mirror the partners section so that the layout is better for smaller devices:

Screenshot 2022-05-10 at 10 06 32

### CODE REVIEW

**Review by Isaiah-Hamilton — APPROVED** [review]
lgtm

File: apps/www/pages/partners/experts/index.tsx [hunk 1]
```diff
@@ -62,61 +62,63 @@ function ExpertPartnersPage(props: Props) {
             <h1 className="h1">{meta_title}</h1>
             <h2 className="text-scale-900 text-xl">{meta_description}</h2>
           </div>
-          <div className="grid grid-cols-12 lg:gap-16 xl:gap-32">
-            <div className="col-span-3">
+          <div className="grid space-y-12 md:gap-8 lg:grid-cols-12 lg:gap-16 lg:space-y-0 xl:gap-16">
```
Comment (Isaiah-Hamilton) [inline apps/www/pages/partners/experts/index.tsx:65]:
```suggestion
          <div className="grid space-y-12 md:gap-8 lg:grid-cols-12 lg:gap-16 lg:space-y-0 xl:gap-32">
```

File: apps/www/pages/partners/experts/index.tsx [hunk 2]
```diff
@@ -62,61 +62,63 @@ function ExpertPartnersPage(props: Props) {
...
-                      >
-                        <path
-                          stroke-linecap="round"
-                          stroke-linejoin="round"
-                          d="M15.232 5.232l3.536 3.536m-2.036-5.036a2.5 2.5 0 113.536 3.536L6.5 21.036H3v-3.572L16.732 3.732z"
-                        />
-                      </svg>
-                    }
-                  />
+                  <div className="grid grid-cols-2 gap-8 lg:grid-cols-1">
```
Comment (Isaiah-Hamilton) [inline apps/www/pages/partners/experts/index.tsx:72]:
```suggestion
                  <div className="grid grid-cols-2 gap-4 lg:grid-cols-1">
```

**Review by Isaiah-Hamilton — CHANGES_REQUESTED** [review]
I have some small changes.

File: apps/www/pages/partners/experts/index.tsx (hunk 2 above)
Comment (Hallidayo) [reply apps/www/pages/partners/experts/index.tsx]: Committed this change in `5fe1522`

File: apps/www/pages/partners/experts/index.tsx (hunk 1 above)
Comment (Hallidayo) [reply apps/www/pages/partners/experts/index.tsx]: Committed this change in `5fe1522`

**Review by Isaiah-Hamilton — APPROVED** [review]

**Pull request merged**